main.go

```diff
@@ -100,7 +102,8 @@ func Main() int {
 		return log.FErrf("Error initializing extensions: %v", err)
 	}
 	if *commandFlag != "" {
-		res, errs, _ := repl.EvalString(*commandFlag)
+		// return EvalStringWithOption(Options{All: true, ShowEval: true, NoColor: true, Compact: false}, what)
```

Comment:
Left behind debug?

Reply:
yeah or rather I copied that one from EvalString to see which options were set before